buildtools/commands/update-readme.js

```diff
@@ -0,0 +1,28 @@
+const { resolve, join } = require('path');
+const shell = require('shelljs');
+const replace = require('replace-in-file');
+
+const ROOT_DIR = resolve(__dirname, '../../');
+
+function getPublishedWidgetVersion () {
+  const stdout = shell.exec('yarn info @okta/okta-signin-widget versions', { silent: true });
```

Comment:
would be easier to check 
```
'dist-tags': {
    latest: '5.3.3',
    legacy: '5.2.4'
  },

```

from `yarn info @okta/okta-signin-widget` ?

Reply:
`npm view ..` gives similar lighter result

Reply:
The output of  `yarn info @okta/okta-signin-widget versions` comes out like this:

```
[
  '1.4.0',
  '1.5.0',
 <snip>
  '3.0.4',
  '5.3.1',
  '5.3.2',
  '5.3.3',
  '5.4.0'
]
```
so is it is easy to parse it as an array and grab the last entry. 

Are you thinking of a case where we want to tag an older version as latest because of a bad release?

Reply:
Could be. and I don't trust the Array :D instead of more **explicit** meta data like what I post above.